app/src/main/java/com/example/tasky/itemdetail/domain/VisitorType.kt

```diff
@@ -0,0 +1,32 @@
+package com.example.tasky.itemdetail.domain
+
+import com.example.tasky.R
+
+sealed class VisitorType(
+    val name: Int,
```

Comment:
@StringRes val name: Int

Important, since we want to only have stringres id's